lua/hbac/utils.lua

```diff
@@ -0,0 +1,14 @@
+M = {}
+
+M.get_buffers = function()
+	return vim.tbl_filter(function(bufnr)
+		local name = vim.api.nvim_buf_get_name(bufnr)
+		local listed = vim.api.nvim_buf_get_option(bufnr, "buflisted")
+		if name ~= "" and listed then
+			return true
+		end
+		return false
+	end, vim.api.nvim_list_bufs())
+end
```

Comment:
This is the filter function from @ditsuke's PR that I wanted to use to standardize the buflist. Aside from the additional check to see if a buffer is in a window, is there any place we shouldn't be using this?